core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -61,6 +61,14 @@ object RawlsApiService extends LazyLogging {
         )
         Sentry.captureException(rollback)
         complete(StatusCodes.InternalServerError -> ErrorReport(rollback))
+      case sql: SQLException =>
```

Comment:
I left the handling for `SQLTransactionRollbackException`s intact, but we should probably consider if we want this type of handling to happen for that as well, or even as a default for exception types we haven't explicitly enumerated here.